user_guide_src/source/libraries/caching.rst

```diff
@@ -280,8 +276,7 @@ Drivers
 File-based Caching
 ==================
 
-Unlike caching from the Output Class, the driver file-based caching
-allows for pieces of view files to be cached. Use this with care, and
+File-based caching allows for pieces of view files to be cached. Use this with care, and
```

Comment:
This statement is a bit strange because the first half of the original statement has been removed. 
Any cache handler can cache pieces of view files.

Reply:
@kenjis I was just trying to correct this part "the driver file-based caching" to "the file-based caching". After your offer I removed the first half of the sentence. I have no suggestion for making the sentence more meaningful. Full sentence or paragraph offers are welcome.

Reply:
```
File-based Caching
==================

This requires a cache directory to be really writable by the application.

Use this with care, and make sure to benchmark your application, as a point can
come where disk I/O will negate positive gains by caching.
```

Reply:
@kenjis Changed. Thanks.